### ISSUE TITLE
Consider setting up GitHub projects for issue ticket management

### ISSUE THREAD

**Issue by Kissaki** [mumble-voip/mumble#3776]
We have a long list of issues and make use of Labels, and sparingly Milestones to manage them.

I think we could get some value out of setting up Projects. Most importantly, it could help us manage and present feature requests separately from other issues.

Projects could be used to manage two kinds of issue categories:

* Feature- and Milestone-specific to manage and present individual tickets states for a better overview compared to milestones
* Label-like categorization of issues

Our project has numerous feature blocks and applications. While we make use of labels, it can still be viewed as an unfitting, too flat categorization and presentation.

We could create separate projects as “focus groups”. For example a UX project for improving the client user experience.

For a feature requests project I am thinking state columns: Suggested (in Triage), Accepted (we would generally accept and/or appreciate an implementation of it), [Planned,] WIP, Implemented.

@davidebeatrici What do you think?

**Comment by davidebeatrici** [comment]
Labels are definitely not enough to split the huge amount of issues.

I like your idea, that would definitely help.

**Comment by Kissaki** [comment]
I plan to create some then after we did the release.

**Comment by davidebeatrici** [comment]
Excellent.

**Comment by Kissaki** [comment]
Closing this as the discussion basically concluded/ended.

A feature request project has been created. But not issues have been applied yet. I created a bot that would auto-add tickets that are labeled as feature request, but it is not set up yet. I also saw that github actions seems to support some more direct ticket/repository management which may be a better approach to automating it.

No projects have been created yet for feature areas. Setting them up would require not just creating them but also adding the corresponding tickets to it. And as long as we don’t work on it, creating tickets to implement it may be more bloat than useful.
